app/views/layouts/mailer.html.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
What's this file for? Just a default?

Reply:
Yes, this is a  default template layout.